eventdata/runners/nodestorage_runner.py

```diff
@@ -44,7 +44,7 @@ async def nodestorage(es, params):
         data_node_count = 0
 
         for node_role in node_role_list:
-            if 'd' in node_role:
+            if 'd' in node_role or 'h' in node_role or 'f' in node_role:
```

Comment:
I'd argue that any node role that is capable of hosting indices should be counted as a data node here regardless whether it would skew results. I'm also proposing a slightly different implementation that allows us to add new node roles more easily if needed. We could write this line instead as:

```python
if node_role in ["c", "d", "f", "h", "w", "s"]:
```

See https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html for a definition of the node roles and https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-nodes.html for the node role names in the `cat nodes` API.

Reply:
makes sense, thank you